test/torchaudio_unittest/datasets/commonvoice_test.py

```diff
@@ -17,73 +17,56 @@
 _ORIGINAL_EXT_AUDIO = COMMONVOICE._ext_audio
 _SAMPLE_RATE = 48000
 _HEADERS = [u"client_ids", u"path", u"sentence", u"up_votes", u"down_votes", u"age", u"gender", u"accent"]
-
-
-def get_mock_dataset_en(root_dir) -> Tuple[Tensor, int, Dict[str, str]]:
+_EN_TRAIN_CSV_CONTENTS = [
+    ["9d16c5d980247861130e0480e2719f448be73d86a496c36d01a477cbdecd8cfd1399403d7a77bf458d211a70711b2da0845c",
+     "common_voice_en_18885784.wav",
+     "He was accorded a State funeral, and was buried in Drayton and Toowoomba Cemetery.", "2", "0", "", "",
+     ""],
+    ["c82eb9291328620f06025a1f8112b909099e447e485e99236cb87df008650250e79fea5ca772061fb6a370830847b9c44d20",
+     "common_voice_en_556542.wav", "Once more into the breach", "2", "0", "thirties", "male", "us"],
+    ["f74d880c5ad4c5917f314a604d3fc4805159d255796fb9f8defca35333ecc002bdf53dc463503c12674ea840b21b4a507b7c",
+     "common_voice_en_18607573.wav",
+     "Caddy, show Miss Clare and Miss Summerson their rooms.", "2", "0", "twenties", "male", "canada"],
+]
+
+_FR_TRAIN_CSV_CONTENTS = [
+    [
+        "a2e8e1e1cc74d08c92a53d7b9ff84e077eb90410edd85b8882f16fd037cecfcb6a19413c6c63ce6458cfea9579878fa91cef"
+        "18343441c601cae0597a4b0d3144",
+        "89e67e7682b36786a0b4b4022c4d42090c86edd96c78c12d30088e62522b8fe466ea4912e6a1055dfb91b296a0743e0a2bbe"
+        "16cebac98ee5349e3e8262cb9329",
+        "Or sur ce point nous n’avons aucune réponse de votre part.", "2", "0", "twenties", "male", "france"],
+    [
+        "a2e8e1e1cc74d08c92a53d7b9ff84e077eb90410edd85b8882f16fd037cecfcb6a19413c6c63ce6458cfea9579878fa91cef18"
+        "343441c601cae0597a4b0d3144",
+        "87d71819a26179e93acfee149d0b21b7bf5e926e367d80b2b3792d45f46e04853a514945783ff764c1fc237b4eb0ee2b0a7a7"
+        "cbd395acbdfcfa9d76a6e199bbd",
+        "Monsieur de La Verpillière, laissez parler le ministre", "2", "0", "twenties", "male", "france"],
+
+]
+
+
+def get_mock_dataset(root_dir, train_csv_contents, ext_audio) -> Tuple[Tensor, int, Dict[str, str]]:
```

Comment:
Could you keep the original `get_mock_dataset_en` and `get_mock_daatset_fr` functions and make them use this `get_mock_dataset`?

That way, the later when a new maintainer comes in and wants to add a new test re-using the existing language, they can just call these functions without knowing the internals.

Something like

```python
def get_mock_dataset(root_dir, train_csv_contents):
    ...

def get_mock_dataset_en(root_dir):
    return get_mock_dataset(root_dir, _EN_TRAIN_CSV_CONTENTS)

def get_mock_dataset_en(root_dir):
    return get_mock_dataset(root_dir, _FR_TRAIN_CSV_CONTENTS)
```

Reply:
Hi @mthrok, happy new year.
Thanks for the detailed feedback.
I have updated my PRs according to your previous comments.